sdk/peering/Azure.ResourceManager.Peering/src/autorest.md

```diff
@@ -86,6 +86,8 @@ rename-mapping:
   BgpSession.peerSessionIPv6Address: -|ip-address
   CdnPeeringPrefix.properties.azureRegion: -|azure-location
   PeeringServiceLocation.properties.azureRegion: -|azure-location
+  RpUnbilledPrefix: RoutingPreferenceUnbilledPrefix
+  RpUnbilledPrefix.AzureRegion: -|azure-location
```

Comment:
It seems the type of the generated code is still string.

Reply:
fixed